src/placement/jump_map.c

```diff
@@ -569,7 +572,8 @@ get_object_layout(struct pl_jump_map *jmap, struct pl_obj_layout *layout,
 			layout->ol_shards[k].po_fseq = fseq;
 
 			/** If target is failed queue it for remap*/
-			if (pool_target_unavail(target)) {
+			if (pool_target_unavail(target) && !(ignore_up == true &&
+                                        target->ta_comp.co_status == PO_COMP_ST_UP))  {
```

Comment:
(style)  line over 80 characters
(style)  code indent should use tabs where possible